anta/tests/cvx.py

```diff
@@ -87,3 +87,75 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyMcsServerMounts(AntaTest):
+    """Verify if all MCS server mounts are in mountStateMountComplete.
```

Comment:
Maybe lets write what it means to be in mountStateMountComplete

Would this makes sense?

```suggestion
    """Verify if all MCS server mounts are in a MountComplete state.
```

Reply:
`mountStateMountComplete` is the exact text that comes in the show command output.